src/main/java/lottery/domain/WinningNumber.java

```diff
@@ -0,0 +1,37 @@
+package lottery.domain;
+
+import lottery.domain.exception.InconsistentLotteryException;
+import lottery.domain.strategy.Picker;
+
+import java.util.Collection;
+import java.util.List;
+
+public class WinningNumber extends Lottery {
+    public static final int COUNT_WINNING_NUMBER = 7;
+
+    private final LotteryNumber bonusNumber;
+
+    public static WinningNumber from(Picker picker) {
+        List<LotteryNumber> candidate = picker.pick(COUNT_WINNING_NUMBER);
+        return new WinningNumber(candidate.subList(0, candidate.size() - 1), candidate.get(candidate.size() - 1));
+    }
+
+    public WinningNumber(Collection<LotteryNumber> numbers, LotteryNumber bonusNumber) {
+        super(numbers);
+        this.bonusNumber = bonusNumber;
+        if(numbers.contains(bonusNumber)) {
+            throw new InconsistentLotteryException();
+        }
+    }
+
+    public WinningType countMatched(Lottery another) {
+        int matches = (int) another.getNumbers()
+                .stream()
+                .filter(getNumbers()::contains)
+                .count();
+        boolean hasBonus = another.getNumbers()
+                .stream()
+                .anyMatch(i -> i == bonusNumber);
```

Comment:
```suggestion
        boolean hasBonus = another.hasNumber(bonusNumber);
```
이런식으로 표현할 수 있게 리팩토링 해보면 어떨까요?